webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -816,9 +818,14 @@ public EntityMutationResponse createOrUpdate(AtlasEntitiesWithExtInfo entities,
         RequestContext.get().setEnableCache(false);
         RequestContext.get().setSkipProcessEdgeRestoration(skipProcessEdgeRestoration);
         try {
+            int entitiesCount = CollectionUtils.isEmpty(entities.getEntities()) ? 0 : entities.getEntities().size();
```

Comment:
If the entities list are null anyway the request is no valid, so if CollectionUtils.isEmpty(entities.getEntities()) return